proxies/go/handler_client.go

```diff
@@ -43,17 +43,17 @@ func clientHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	options := devcycle.DVCOptions{
-		ConfigCDNOverride:     reqBody.Options.ConfigCDNOverride,
-		EventsAPIOverride:     reqBody.Options.EventsAPIOverride,
+		ConfigCDNURI:     reqBody.Options.ConfigCDNOverride,
```

Comment:
i think the names of reqBody.Options should be changed too but i'll do that in my pr with the localbucketing change